### ISSUE TITLE
Expose ws.onclose method to client

### ISSUE THREAD

**Issue by sOm2y** [onmax/albatross-rpc-client-ts#67]
when there aren't many events on subscribed address the websocket seems to close the session probably due by default os/firewall/network setting tcp keep alive: 7200 - 2 hours 
For example, I use this function to monitor/subscribe my block rewards when my validator has been elected in current epoch, I can see events coming in but when it's not elected and you have to wait 12 hours to be elected again, the websocket timed out already, have to manually restart the process.

```
const { next, close } = await client.blockchainStreams.subscribeForLogsByAddressesAndTypes(
      {
        addresses: [validatorAddresss],
        types: [], 
        withMetadata: true,
      },
      { once: false }
    )
```

proposed way to mitigate that:
expose `ws.onclose` to client, so we can pass callback function to reset the subscribe function

```
   ws.onclose = function(){
        setTimeout(subscribeFunctionAgain, 1000);
    };
```

**Comment by onmax** [comment]
Hey, just released `v0.5.5`. ea39457214465267ba7d921408f8f09c8995d466

Now you should be able to access the `WebSocket` instance as follows

```ts
const { next, close, ws } = await client.blockchainStreams.subscribeForLogsByAddressesAndTypes(...)

ws.onclose()
```

Let me know if this solution works for you, specially if you find any issues with the types. Thank you!

PS: I am planning to refactor the WebClient implementation to use a WebSocket library to take advantage of features like auto-reconnect, timeout... You can follow #64 to find out when this will be available :)

**Comment by sOm2y** [comment]
> Hey, just released `v0.5.5`. [ea39457](https://github.com/onmax/albatross-rpc-client-ts/commit/ea39457214465267ba7d921408f8f09c8995d466)
> 
> Now you should be able to access the `WebSocket` instance as follows
> 
> ```ts
> const { next, close, ws } = await client.blockchainStreams.subscribeForLogsByAddressesAndTypes(...)
> 
> ws.onclose()
> ```
> 
> Let me know if this solution works for you, specially if you find any issues with the types. Thank you!
> 
> PS: I am planning to refactor the WebClient implementation to use a WebSocket library to take advantage of features like auto-reconnect, timeout... You can follow #64 to find out when this will be available :)

thanks @onmax great work!
